### DISCUSSION TITLE
Linux中怎么配置

### DISCUSSION THREAD

**Discussion by ChuangYu1897** [BeyondDimension/SteamTools#1180] [Q&A] [answered]
装的无界面的Linux，在下载deb文件安装之后怎么对软件进行配置啊？还是不需要配置有没有使用说明之类的文件

**Comment by AigioL** [comment, marked as answer]
目前只能在有 GUI 的 Linux 上运行，通过 UI 进行配置，不支持命令行或配置文件之类的使用

**Reply by ChuangYu1897** [reply]
好的，感谢。不知道后期会不会添加命令行配置之类的
